salt/metalk8s/kubeadm/init/preflight/recommended.sls

```diff
@@ -1,8 +1,17 @@
 {%- from "metalk8s/map.jinja" import kubeadm_preflight with context %}
+{%- from "metalk8s/map.jinja" import repo with context %}
+
+include:
+  - metalk8s.repo
 
 Install recommended packages:
   pkg.installed:
```

Comment:
Yeah, I was not sure about that, since we have no version or repository informations about these packages in the pillar or defaults.yaml, only a list, it's a bit useless to use the macro.